operations/helm/tests/metamonitoring-values-generated/mimir-distributed/templates/metamonitoring/mixin-alerts.yaml

```diff
@@ -841,6 +841,15 @@ spec:
             labels:
               reason: consecutive-failures
               severity: critical
+          - alert: MimirCompactorHasRunOutOfDiskSpace
+            annotations:
+              message: Mimir Compactor {{ $labels.pod }} in {{ $labels.cluster }}/{{ $labels.namespace }} has run out of disk space.
+              runbook_url: https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#mimircompactorhasrunoutofdiskspace
+            expr: |
+              increase(cortex_compactor_disk_out_of_space_errors_total{}[24h]) >= 1
```

Comment:
Using 24h range will make this alert last at least 24h after first occurrence. Is there a way we could make it smarter?

Reply:
I don't have a better idea than to reduce the `24h` range vector, do you have a specific idea?
I think reducing it should be fine actually, as long as it triggers the alert at least once on each occurrence I guess it makes no big difference how far it looks back into the past.

Reply:
I think using 5 or 15 mins here would be fine as well. Even if alert stops firing after 15mins because no compaction failed recently, it will fire again if there's still a problem with the same job and this job is retried.

Reply:
reduced it to 15min https://github.com/grafana/mimir/pull/8278/commits/eff941bfa2bf6dc7c58d3fed5cf41737d0b262f8